File: web/src/models/DTOs/Produtos/ProdutoExternoDTO.ts
export interface ProdutoExternoDTO {
  id: number;
  idProduto: number;
  idSistema: number;
  nome: string;
  status: "A" | "F";
  produtoProtheus: string;
  dataInclusao: string;
  idOperadorInclusao: number;
  dataAlteracao: string;
  idOperadorAlteracao: number;
}

File: webservices/src/main/java/fieg/modulos/cadastro/produtoexterno/dto/ProdutoExternoDTO.java
package fieg.modulos.cadastro.produtoexterno.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class ProdutoExternoDTO {

    private Integer id;
    private Integer idProduto;
    private Integer idSistema;
    private String nome;
    private Character status;
    private String produtoProtheus;
    private LocalDateTime dataInclusao;
    private Integer idOperadorInclusao;
    private LocalDateTime dataAlteracao;
    private Integer idOperadorAlteracao;
}
